attic/multibody/test/contact_toolkit_test.cc

```diff
@@ -121,38 +123,49 @@ class ContactToolkitTest : public ::testing::Test {
 
 // Tests the contact toolkit constructor initializations, getters and setters
 TEST_F(ContactToolkitTest, InitializationTest) {
+  // ContactInfo default values
+  ContactInfo default_contact;
+  ASSERT_TRUE(default_contact.xA.isApprox(Matrix3Xd::Zero(3, 1)));
+  ASSERT_EQ(default_contact.num_contacts, 0);
+
+  // ContactInfo construction validity check
+  Matrix3Xd xa = Matrix3Xd::Zero(3, 2);
+  vector<int> idxa(2, 1);
+  ContactInfo tmp_info = {xa, idxa};
```

Comment:
Can a more descriptive name than tmp_info be used?

Reply:
Yeah will do.